tests/conftest.py

```diff
@@ -151,6 +167,19 @@ def check_cleanup_objects(db):
     ), 'Some objects created in the test need to be cleaned up'
 
 
+# @pytest.fixture(autouse=True)
```

Comment:
Any reason this is left here if it's not used?

Reply:
This has been removed